apparelo/apparelo/doctype/item_production_detail/item_production_detail.py

```diff
@@ -394,6 +394,29 @@ def create_process_details(self):
 				continue
 		if self.additional_flows!=[]:
 			ipd=additional_process(self,ipd)
+		if self.is_combined_packing:
+			input_items_=[]
+			process_variants = {}
+			for ipd_ in self.combined_ipd:
+				ipd_item_mapping = frappe.db.get_value("IPD Item Mapping", {'item_production_details': ipd_.ipd})
+				ipd_doc=frappe.get_doc("Item Production Detail",ipd_.ipd)
+				variants = frappe.get_doc('IPD Item Mapping', ipd_item_mapping).get_process_variants("Ironing")
```

Comment:
- get_doc directly. Instead of, getting a name and then getting doc in the next step.
- remove the hardcoding of "Ironing" as discussed.